Userland/Libraries/LibGUI/VimEditingEngine.cpp

```diff
@@ -657,7 +657,7 @@ void VimMotion::calculate_find_range(VimCursor& cursor, int amount)
     m_find_mode = FindMode::None;
 }
 
-Optional<TextPosition> VimMotion::get_position(VimEditingEngine& engine)
+Optional<TextPosition> VimMotion::get_position(VimEditingEngine& engine, bool can_go_over_endline)
```

Comment:
How about something like `bool in_visual_mode` instead? That makes the intentions more clear.